heartbeat/tests/system/config/heartbeat.yml.j2

```diff
@@ -1,3 +1,5 @@
+logging.level: debug
```

Comment:
This should not have any impact as `-e` is used to capture the logs from stdout.

Reply:
@andrewvc I would suggest to remove the line but don't want to break your tests (it shouldn't  ;-) ). I worry that this might cause confusion in the future.